### ISSUE TITLE
Validation of pre-commit breaks if 'files' property not present for `always_run` hooks

### ISSUE THREAD

**Issue by prat0318** [pre-commit/pre-commit#502]
As per the docs, `files` is optional. But, looks like it is not.

```python
An unexpected error has occurred: ValidationError: u'files' is a required property

Failed validating u'required' in schema:
    {u'properties': {u'additional_dependencies': {u'items': {u'type': u'string'},
                                                  u'type': u'array'},
                     u'always_run': {u'default': False,
                                     u'type': u'boolean'},
                     u'args': {u'default': [],
                               u'items': {u'type': u'string'},
                               u'type': u'array'},
                     u'description': {u'default': u'',
                                      u'type': u'string'},
                     u'entry': {u'type': u'string'},
                     u'exclude': {u'default': u'^$', u'type': u'string'},
                     u'files': {u'type': u'string'},
                     u'id': {u'type': u'string'},
                     u'language': {u'type': u'string'},
                     u'language_version': {u'default': u'default',
                                           u'type': u'string'},
                     u'minimum_pre_commit_version': {u'default': u'0.0.0',
                                                     u'type': u'string'},
                     u'name': {u'type': u'string'},
                     u'stages': {u'default': [],
                                 u'items': {u'type': u'string'},
                                 u'type': u'array'}},
     u'required': [u'id', u'name', u'entry', u'language', u'files'],
     u'type': u'object'}

On instance:
    {'always_run': True,
     'entry': "bash -c 'foo bar'",
     'id': 'swagger-validate',
     'language': 'system',
     'name': 'swagger-validate'}
Check the log at ~/.pre-commit/pre-commit.log
```

**Comment by asottile** [comment]
Workaround for now is to supply a `files: ''` dummy value

**Comment by MikeVL** [comment]
```
> pre-commit 0.13.5

> An error has occurred: InvalidManifestError:
> ==> File /Users/mike/.pre-commit/repo58KPGj/.pre-commit-hooks.yaml
> ==> At Hook(id='no-commit-to-branch')
> =====> Expected files to be absent when always_run is not False, found files: '.*'
> Check the log at ~/.pre-commit/pre-commit.log
```

**Comment by asottile** [comment]
Oops, I didn't intend to break this in the other direction - - I'll release a patch that allows the unnecessary `files` once I'm at a computer.

**Comment by asottile** [comment]
This has been fixed in 0.13.6, sorry for the inconvenience!
